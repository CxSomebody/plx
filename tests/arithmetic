var
  a, b: integer;
  c, d: char;
  t: integer;
begin
  a := 3000;
  b := 1000;
  c := 30;
  d := 10;
  write(a+b);
  write(a-b);
  write(a*b);
  write(a/b);
  t := c+d;
  write(t);
  t := c-d;
  write(t);
  t := c*d;
  write(t);
  t := c/d;
  write(t);
end.
