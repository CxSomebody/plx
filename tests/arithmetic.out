4000
2000
3000000
3
40
20
44
3
